packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -252,7 +253,12 @@ function newClick(
   const id = generateUUID()
   const startClocks = clocksNow()
   const historyEntry = history.add(id, startClocks.relative)
-  const eventCountsSubscription = trackEventCounts(lifeCycle)
+  const eventCountsSubscription = trackEventCounts({
+    lifeCycle,
+    predicate: (event) =>
+      event.action !== undefined &&
+      (Array.isArray(event.action.id) ? includes(event.action.id, id) : event.action.id === id),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏To be a bit more explicit what do you think of naming the predicate functions. Something like `predicate: isChildEvent`